test/simplifyExpression/collectAndCombineSearch/collectAndCombineSearch.test.js

```diff
@@ -10,6 +10,21 @@ function testSimpleCollectAndCombineSearch(exprString, outputStr) {
   TestUtil.testSimplification(collectAndCombineSearch, exprString, outputStr);
 }
 
+describe('combineNthRoots multiplication', function() {
+  const tests = [
+    ['nthRoot(x, 2) * nthRoot(x, 2) * nthRoot(x, 3)',
```

Comment:
Can you add a test case that is just `nthRoot(x, 2) * nthRoot(x, 2)` to make sure it works if there is no collecting, but only combining?